### PR TITLE
Selected points

### PR DESCRIPTION
**Pull request by tsublette**

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -3983,3 +4004,25 @@ definitions:
       description:
         type: string
         description: If the estimate is 0, the description field will contain information on why an estimate could not be calculated
+
+  SelectedPoint:
+    description: describes a point
+    required:
+      - label
```
Comment (ryanwalls) [inline swagger.yaml]: Just curious, but why would we require the label?

File: swagger.yaml [hunk 2]
```diff
@@ -3261,11 +3264,29 @@ definitions:
       bladeCrashThreshold:
         type: number
         description: should be a number between 0.5 and 1.5
+      outputPointThermalHistory:
+        type: boolean
+      outputVirtualSensorData:
+        type: boolean
+      staticVirtualSensorRadius:
+        type: number
+        description: should be a number between 0.05 and 5.0 mm
```
Comment (ryanwalls) [inline swagger.yaml]: Would be good to add more description to these fields.  I can't tell what they are or what the difference between static and dynamic is.

File: swagger.yaml [hunk 3]
```diff
@@ -3261,11 +3264,29 @@ definitions:
       bladeCrashThreshold:
         type: number
         description: should be a number between 0.5 and 1.5
+      outputPointThermalHistory:
+        type: boolean
+      outputVirtualSensorData:
+        type: boolean
```
Comment (ryanwalls) [inline swagger.yaml]: What does it mean when this is true?

File: swagger.yaml (hunk 1 above)
Comment (tsublette) [inline swagger.yaml]: There will be a zipped file per selected point. I hope to use the label in the output so the user will be able to correlate selected point to output

**Pull request merged**